lib/email_record.js

```diff
@@ -11,15 +11,17 @@ module.exports = function (limits, now) {
 
   function EmailRecord() {
     this.xs = []
+    this.ub = []
   }
 
   EmailRecord.parse = function (object) {
     var rec = new EmailRecord()
     object = object || {}
     rec.bk = object.bk       // timestamp when the account was banned
     rec.rl = object.rl       // timestamp when the account was rate-limited
-    rec.xs = object.xs || [] // timestamps when emails were sent
+    rec.xs = object.xs || rec.xs // timestamps when emails were sent
```

Comment:
I just noticed that an Array is already initialized in the constructor, why create a new one. One might hope that V8 can initialize arrays for nothing, but :shrug: